### DISCUSSION TITLE
Alt Actions

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#45] [Announcements]
Commands now can have alternative actions when triggered by a middle click.
Color changers like `Color Red` have the alt action to remove colors.
So you can click the red button in the favorites bar to color red, and middle click it to remove color.
That's the only alt action for now.

**Comment by N3C2L** [comment]
Really good improvement, thanks.

**Reply by madprops** [reply]
Decided to remove this. It complicated stuff.
